src/app/retrospective/sprint-task-summary/sprint-task-summary.component.ts

```diff
@@ -539,13 +558,42 @@ export class SprintTaskSummaryComponent implements OnInit, OnChanges, OnDestroy
                     },
                 },
             },
+            ...deleteButtonColumnDef,
         ];
     }
 
     getDisplayedRowCount() {
         return (this.gridApi && this.gridApi.getDisplayedRowCount()) || 0;
     }
 
+    deleteTask(params: any) {
+        const task = params.data;
+        const dialogRef = this.dialog.open(BasicModalComponent, {
+            data: {
+                content: 'Are you sure you want to delete ' + (task.Key) + ' task?',
```

Comment:
Please avoid `+` if possible.

```suggestion
                content: 'Are you sure you want to delete the task' + (task.Key) + ' ?',
```

Reply:
done.